docs/calicoctl/endpoint.md

```diff
@@ -0,0 +1,191 @@
+|  calico  |      docker     | 0d01b3f020fcadfd0090fcbbbbef9658acb26f71c1cb812827afafc625c5ae1a | d79123c4784511e5bd1a080027f532f6 | 192.168.1.4/32 | d6:43:59:f7:93:d3 |          | active |
+|  calico  |      docker     | 26d5636108f8e46b0b9d663522c26f05e52acecf11bb10104dfbd9047ad502b6 | d50c35bc784511e5bd1a080027f532f6 | 192.168.1.3/32 | 0e:61:2a:84:30:51 |   PROF   | active |
+|  calico  |      docker     | 838a52d2809845a1092c9cb0f8ff5a09437339cbd39b597924eaefacbf50b4ae | d0bd03d8784511e5bd1a080027f532f6 | 192.168.0.1/32 | b6:bc:87:55:54:a0 |   PROF   | active |
+|  calico  |      docker     | b75d50055975385f0f26ecfc545e91c10a911dbfff4894a03510a617c4b232fc | d2b58ef8784511e5bd1a080027f532f6 | 192.168.1.2/32 | f2:42:82:f6:1f:a4 |   PROF   | active |
+|  calico  |      docker     | ccc8d2e81b8ef7d4eeb8634e61ed515da47f7d5d7803f40408af21c002f379db | d9d6c3a0784511e5bd1a080027f532f6 | 192.168.1.5/32 | f6:ed:d8:c5:dd:ac |          | active |
++----------+-----------------+------------------------------------------------------------------+----------------------------------+----------------+-------------------+----------+--------+
+
+$ calicoctl endpoint show --endpoint=d79123c4784511e5bd1a080027f532f6 --detailed
++----------+-----------------+------------------------------------------------------------------+----------------------------------+----------------+-------------------+----------+--------+
+| Hostname | Orchestrator ID |                           Workload ID                            |           Endpoint ID            |   Addresses    |        MAC        | Profiles | State  |
++----------+-----------------+------------------------------------------------------------------+----------------------------------+----------------+-------------------+----------+--------+
+|  calico  |      docker     | 0d01b3f020fcadfd0090fcbbbbef9658acb26f71c1cb812827afafc625c5ae1a | d79123c4784511e5bd1a080027f532f6 | 192.168.1.4/32 | d6:43:59:f7:93:d3 |          | active |
++----------+-----------------+------------------------------------------------------------------+----------------------------------+----------------+-------------------+----------+--------+
+
+```
+
+### calicoctl endpoint <ENDPOINT_ID> profile (append|remove|set)
+> NOTE: This command should NOT be used when running Calico with the Docker 
+> libnetwork driver.  The libnetwork driver manages the security profiles for 
+> containers. 
+
+This command is used to manage security profiles associated with endpoints. 
+
+The command allows you to:
+ - append profiles to the end point by adding any passed in profiles to the 
+ list of profiles already associated with the endpoint
+ - remove profiles from the endpoint's list of profiles
+ - set a list of profiles that replaces the current list of profiles associated 
+ with the endpoint.
+
+The more common command for profile management with containers is the 
```

Comment:
I would qualify this by saying "If you are controlling Calico network policy for Docker containers using the default networking (i.e. not libnetwork), it is more common to..."